shopfloor_dangerous_goods_mobile/static/wms/src/css/dangerous_goods.css

```diff
@@ -0,0 +1,10 @@
+.detail-picking-select .has-lq-products .v-list-item__title::after {
```

Comment:
Note to myself: this selector is compliant w/ a change that I still have to push in another PR

Reply:
Here's the change https://github.com/OCA/wms/pull/582

Reply:
merged, rebased, we are good to go